app/assets/stylesheets/layout/_layout.scss

```diff
@@ -30,6 +33,25 @@ body {
     }
   }
 
+  .box{
+    padding: 3em;
+    background: lighten($light-gray, 11%);
+    border-radius: $base-border-radius;
+    box-shadow: inset 0 0 1px $light-gray, 0 2px 4px darken($base-background-color, 10%);
+  }
+
+  .authors{
+    a{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space